### ISSUE TITLE
Normative Language on Registration Policy

### ISSUE THREAD

**Issue by henkbirkholz** [ietf-wg-scitt/draft-ietf-scitt-architecture#159]
Currently in 24f63ee8cfc2b99b937dd236cca3c886fd1218f9, the Introduction includes normative language about Registration Policies.

> Each service MAY enforce its own Registration Policies for authorizing entities to register their Signed Statements to the Append-only Log.

Independent of where this normative language goes in the end, I'd assume that a MAY is too weak? The minimal required registration policy - from my individual point of view - seems to be the authorization of the Issuer that is the "Transparency Services Owner" to register new Signed Statements. Does that sound right? (in essence that roughly resembles a "key before signatures" approach to ensure that a transparency service retains the capability to register Signed Statements).

If there always must be a Minimum of an initial Registration Policy (and be it part of bootstrapping a TS with its first statement that then must be a policy including a key), does this normative language not require a MUST?

What does the group think. Am I missing something obvious here?

**Comment by henkbirkholz** [comment]
The current rough consensus seems to lean towards putting Registration Policies out-of-scope of the core SCITT Architecture (with the exception of the "key before signature" concept). If there is no objection in the next days, I'll start a PR after #163 is resolved.

**Comment by SteveLasker** [comment]
Completed in 05. Need specific, new issues.
